docs/docs/canonicai/chapter9.md

```diff
@@ -0,0 +1,33 @@
+# Stand up a jaseci_ai_kit server locally
+
+In this tutorial, we will discuss how to use uvicorn to stand up a jaseci_ai_kit server locally and quickly
+
+Are you Excited? Let's jump right in.
+
+# Process
+First, you will have to cd into the path of the jaseci_ai_kit module for example let try to stand up cl_summer AI module. From the root jaseci directory run this command in the terminal as follows:
```

Comment:
Hey @Shawn-Jemmott l suggest you do a round of proof read after initial draft for this and any future writing contributions. Let's make sure there aren't any grammar issues and sentences and punctuations are used properly etc.